install_config/configuring_local.adoc

```diff
@@ -475,4 +476,4 @@ spec:
 
 [NOTE]
 ====
-The volume is not mounted in the pod but is exposed as the *_/dev/xvda_* raw block device.
\ No newline at end of file
+The volume is not mounted in the pod but is exposed as the *_/dev/xvda_* raw block device.
```

Comment:
@tmorriso-rh - there is a missing ==== here that is causing the build errors. If you fix that as well it should work.